### PR TITLE
Fix double negatives used with the word 'neither' in docs and comments

### PR DESCRIPTION
**Pull request by lanurmi**
In many cases it should be 'either', and 'nor' should be 'or' accordingly.

No changes to actual code.

### CODE REVIEW

File: tests/test.cpp [hunk 1]
```diff
@@ -404,7 +404,7 @@ static int TestCrtReportHook(int reportType, char *message, int *)
 int main(int argc, char **argv)
 {
     // tests can be ran non-interactively so make sure we don't show any assert
-    // dialog boxes -- neither our own nor from MSVC debug CRT -- which would
+    // dialog boxes -- either our own nor from MSVC debug CRT -- which would
```
Comment (vadz) [inline tests/test.cpp:407]: I'm almost sure this is incorrect, "either ... nor" doesn't really work, does it?

**Review by vadz — COMMENTED** [review]
Most of the changes here are definitely correct, thanks, but while I didn't have time to look at all of them, randomly selecting the first and the last one I'm already not sure about the latter, so it would be really great if somebody (preferably, but not necessarily, a native English speaker) could go over all the changes and double check them. TIA!

File: tests/test.cpp (hunk 1 above)
Comment (swt2c) [reply tests/test.cpp:407]: Yes, this one is correct as-is.

Comment (lanurmi) [reply tests/test.cpp:407]: Agreed, there was an error here, and the change has been undone. Please pick another two random changes.

**Pull request closed without merging**